README.md

```diff
@@ -26,24 +28,10 @@ Here is how to configure the Compute Engine firewall.
 
 Now you're good to go!
 
-To build:
-
-1. Rewrite the value of the `application` element in your `appengine-web.xml` to your app id.
-2. Run `mvn package`
-3. Run `appcfg.sh` of the SDK as follows:
-
-        $ $SDK_DIR/bin/appcfg.sh -s preview.appengine.google.com update target/websocketchat-1.0-SNAPSHOT
-
-4. Visit `http://chat.your-app-id.appspot.com/`.
-
-For further information, consult the [Java App
-Engine](https://developers.google.com/appengine/docs/java/overview)
-documentation.
-
-To see all the available goals for the App Engine plugin, run
+##Deploy
 
-    mvn help:describe -Dplugin=appengine
+After setting up Maven, you can deploy your app locally, or to the Google Cloud Platform as described [here](https://cloud.google.com/appengine/docs/java/managed-vms/maven#run_and_deploy_your_app_with_the_cloud_sdk_development_server), then visit `http://chat.your-app-id.appspot.com/`.
 
-[1]: https://docs.google.com/document/d/1VH1oVarfKILAF_TfvETtPPE3TFzIuWqsa22PtkRkgJ4
-[2]: https://groups.google.com/forum/?fromgroups#!topic/google-appengine/gRZNqlQPKys
+[1]: https://cloud.google.com/appengine/docs/java/managed-vms/
+[2]: https://cloud.google.com/appengine/docs/java/managed-vms/#install-sdk
 [3]: https://cloud.google.com/console
```

Comment:
Is it possible to update the cloud console link to:
https://console.developers.google.com/
?

Although I'm OK for me to create another CL for this.

It's off-topic, but I think we'd better automate this kind of change.